az_quickstart.info.yml

```diff
@@ -59,6 +59,7 @@ install:
   - drupal:toolbar
   - drupal:field_ui
   - drupal:file
+  - drupal:flag
```

Comment:
Based on the discussion in our 2023-04-28 Friday meeting, it sounds like we don't want the module to be installed by default.
```suggestion
```